app/search/classification/domain.py

```diff
@@ -0,0 +1,23 @@
+from urllib.parse import urlparse
+
+from .base import BaseClassifier
+
+
+class DomainClassifier(BaseClassifier):
+    """
+    Classifier that is applied if the returned result is the root address of a
+    domain name. Adds:
+
+    fqdn - the fully qualified domain name
+    logo - the URL to a logo, as determined by Clearbit.
```

Comment:
It seems like the list of services is buried in the classifiers, might be nice to reify them. What if we separated out some really, really simple service facade objects? That would make it easier to unit test the classifiers, and would also decouple services from classifiers.